packages/terra-consumer-nav/src/components/nav-logo/NavLogo.jsx

```diff
@@ -43,10 +43,17 @@ const NavLogo = ({
 }) => {
   const imgProps = {
     className: cx('img'),
-    ...url && { src: url },
+    ...url ? {
+      src: url,
+      alt: altText,
+    } : {
+      children: altText,
+    },
   };
 
-  const imgTag = (<img alt={altText} {...imgProps} />);
+  const LogoElement = url ? 'img' : 'p';
```

Comment:
instead of a <p> should it just be null?

Reply:
imgTag has to be something since that's what's being displayed in the smartlink, figured a 'p' tag made the most sense for text